package.json

```diff
@@ -29,31 +33,48 @@
   },
   "homepage": "https://github.com/FabricLabs/portal-feed#readme",
   "dependencies": {
-    "@fabric/core": "FabricLabs/fabric#security/dependencies",
-    "@fabric/http": "FabricLabs/fabric-http#feature/rollup"
+    "@fabric/core": "FabricLabs/fabric#feature/federation",
+    "@fabric/http": "FabricLabs/fabric-http#feature/federation"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.17.12",
+    "@observablehq/plot": "^0.5.0",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-commonjs": "^22.0.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
     "@rollup/plugin-url": "^7.0.0",
+    "d3": "^7.4.4",
+    "d3-graphviz": "^4.1.1",
     "fomantic-ui": "^2.8.8",
     "fomantic-ui-css": "^2.8.8",
     "http-server": "^14.1.0",
+    "lodash.merge": "^4.6.2",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
     "rollup": "^2.74.1",
     "rollup-plugin-import-css": "^3.0.3",
     "rollup-plugin-livereload": "^2.0.5",
+    "rollup-plugin-polyfill-node": "^0.9.0",
     "rollup-plugin-serve": "^1.1.0",
     "semantic-ui-react": "^2.1.2"
   },
+  "peerDependencies": {
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0"
+  },
```

Comment:
I think you might have some success with putting React + other externals in peer dependencies for other projects